checkBalanced.js

```diff
@@ -0,0 +1,96 @@
+class TreeNode {
```

Comment:
Although whitespace doesn't technically matter in JS, the convention for this language is to use 2 spaces, not 4.